test_env_vars.yaml

```diff
@@ -244,3 +244,8 @@ various_packages:
     repo: https://github.com/cachito-testing/cachito-gomod-test
     ref: 1827221d787cbd1e979b339cfbbf59728eddf0d4
     dependencies_count: 48
+# Test data for the private repos test
+# The test will only be performed if $JOB_NAME includes one of the following
+private_repo_test_envs:
+  - cachito-prod
+  - cachito-stage
```

Comment:
Can we have this data elsewhere please? I believe [tests/integration/test_data/private_repo_packages.yaml](https://github.com/containerbuildsystem/cachito/pull/779/files#diff-c0f35f02e13615fcf7b7925d912b94e8299aaf78c70b071d0591f12438f5ee8d) should be fine for that. We wanted to move all specific test data from this file because we have to change it also downstream every time we make a change here.

Reply:
Moved to the private repo test data file :+1: